public/angular-app/settings/settings.html

```diff
@@ -273,488 +274,50 @@ <h3 class="panel-title">File Path Info</h3>
-                                    </div>
+                            <div class="col-sm-12"
+                                 style="margin-top: 10px;"
+                                 ng-repeat="(key1, value1) in value">
+                                <div class="col-sm-3"></div>
+                                <div class="col-sm-9 no-padding-right">
                                     <div class="col-md-3 no-padding-left">
-                                        <input id="infoLocationMatch"
-                                               type="number"
-                                               placeholder="Pattern 'Match' Number"
-                                               class="form-control input-sm"
-                                               ng-model="vm.settings.projectInfo.projectNumber.bimThreeSixty.match">
-                                    </div>
-                                    <div class="col-md-3 no-padding-left">
-                                        <input id="infoLocationGroup"
-                                               type="number"
-                                               placeholder="Pattern 'Group' Number"
-                                               class="form-control input-sm"
-                                               ng-model="vm.settings.projectInfo.projectNumber.bimThreeSixty.group">
+                                        <label class="control-label text-right pull-right"
+                                               style="margin-top: 8px;"
+                                               for="userLocation">
+                                            {{vm.processPropertyName(key1)}}
+                                            <button class="fa-wrapper"
+                                                    ng-if="vm.isValidRegex(value1.pattern)">
+                                                <i class="fas fa-check fa-sm"
+                                                   style="color: green;"></i>
+                                            </button>
+                                            <button class="fa-wrapper"
+                                                    ng-if="!vm.isValidRegex(value1.pattern)">
+                                                <i class="fas fa-asterisk fa-sm"
+                                                   style="color: red;"></i>
+                                            </button>
+                                        </label>
+                                    </div>
+                                    <div class="col-md-9 no-padding">
+                                        <div class="col-md-6 no-padding-left">
+                                            <input id="infoLocationPattern"
+                                                   type="text"
+                                                   placeholder="Location Regex"
```

Comment:
Is 'Location' in reference to Project Location / Project Name / Project Number or just a generic reference to filepath location? If the former, `placeholder="{{vm.processPropertyName(key)}} Regex"` should work here.